packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -262,11 +264,18 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
       // used on single selection to open the dropdown with the selected option as highlighted.
       defaultHighlightedIndex: this.props.multiple ? undefined : null,
       focused: false,
+      highlightedIndex: null,
       searchQuery: search ? '' : undefined,
       value: multiple ? [] : null,
     }
   }
 
+  componentWillReceiveProps(newProps: DropdownProps) {
```

Comment:
!!! `componentWillReceiveProps` is **deprecated** and will be removed in future versions
Consider using `componentDidUpdate`, see:
https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops